packages/palette/src/elements/Cards/MediumCard.tsx

```diff
@@ -1,10 +1,50 @@
 import React from "react"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Image } from "../Image"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { CardTag } from "./CardTag"
 import { MediumCardProps } from "./MediumCard.shared"
 
 /**
  * `MediumCard` is a card with one image one tall image, and text for title and subtitle
  * at the bottom.
  */
-export const MediumCard: React.FC<MediumCardProps> = ({}) => {
-  return null
+export const MediumCard: React.FC<MediumCardProps> = ({
+  image,
+  title,
+  subtitle,
+  tag
+}) => {
+  return (
+    <Box width="280px" height="370px" position="relative">
```

Comment:
I know the specs say this is a fixed size but I don't see how that'd work in practice (vs say a scalable aspect ratio box) — why the fixed px size?

Reply:
@dzucconi - I believe the idea here is that this card should be used in the top carousel of featured works on this page: https://www.figma.com/file/8k9EDGkQKclWjFwoyP8tuJ/Artsy-Viewing-Room-Landing-Page-%5BWeb%5D?node-id=82%3A0 and this carousel is fixed height.

Reply:
After talking with anson, I think we can merge and we can change it later if it's needed, but it seems fixed is what we want here.